src/RestControllers/AuthorizationController.php

```diff
@@ -114,15 +114,16 @@ public function __construct($providerForm = true)
                 http_response_code(500);
                 exit;
             }
-            // second, create and store the private/public key pain
-            $keys = openssl_pkey_new(["private_key_bits" => 2048, "private_key_type" => OPENSSL_KEYTYPE_RSA]);
+            // second, create and store the private/public key pair
+            $keys = \openssl_pkey_new(["default_md" => "sha256", "private_key_bits" => 2048, "private_key_type" => OPENSSL_KEYTYPE_RSA]);
             if ($keys === false) {
                 // if unable to create keys, then force exit
                 error_log("OpenEMR error - key generation broken during oauth2, so forced exit");
                 http_response_code(500);
                 exit;
             }
-            openssl_pkey_export($keys, $privkey, $this->passphrase);
+            $privkey = '';
+            openssl_pkey_export($keys, $privkey);
```

Comment:
@sjpadgett , i just noted this line. Note that this will remove the passphrase protection of the private key.